http-ballerina-tests/build.gradle

```diff
@@ -212,3 +216,35 @@ publishing {
         }
     }
 }
+
+task extractBallerinaClassFiles(type: Copy) {
+    def jars = fileTree(artifactJar).findAll {
+         it.name.endsWith(".jar")
+    }
+
+    jars.forEach { file ->
+        println(file.name)
```

Comment:
do we need this print ?

Reply:
We don't need. I will remove it

Reply:
commit ID [#29d784c](https://github.com/ballerina-platform/module-ballerina-http/pull/230/commits/29d784cbdbba0b9a63e304186dcb640a977da7e5)